api/queries.py

```diff
@@ -9,7 +9,8 @@ class Query(graphene.ObjectType):
 
 class Mutation(graphene.ObjectType):
 	create_user = CreateUser.Field()
-	auth = AuthMutation.Field()
+	# auth = AuthMutation.Field()
```

Comment:
It looks like you have moved the access token into the new sign-in endpoint.   Can this commented out line be removed?